MarkBind/markbind
# Finalize migration to Vue 2.0

**What is the purpose of this pull request? (put "X" next to an item, remove the rest)**

• [X] Other, please explain: Migration



Follows #245.
Fixes #274.



**What is the rationale for this request?**
We need to migrate to a new version of Vue as part of the migration of components to bootstrap-vue.

**What changes did you make? (Give an overview)**
* Migrate the documentation over from vue-strap to MarkBind.
* Push a commit that updates Vue from `1.0` to `2.0`
* Push a commit that updates vue-strap from `v1.1.37-markbind.11` to `v2.0.0-markbind.12`

**Provide some example code that this change will affect:**
N.A. If something breaks, something is wrong with the migration.

**Is there anything you'd like reviewers to focus on?**
Testing that nothing breaks.

**Testing instructions:**
1. Update to Vue 2.0 and incoming vue-strap v2.0.0-markbind.12.
  a. Copy `dist/vue-strap.min.js` from vue-strap repository to MarkBind repository (`assets/js`).
  b.Copy `node_modules/vue/dist/vue.min.js` from vue-strap repository to MarkBind repository (`assets/js`) in order to upgrade to Vue 2.0.
2. Clone https://github.com/nus-cs2103/website-base.git.
3. Run `markbind serve`. No errors should occur.
4. Eye-ball anything to ensure that nothing is broken.
